frontend/src/concepts/k8s/K8sNameDescriptionField/K8sNameDescriptionField.tsx

```diff
@@ -118,8 +121,10 @@ const K8sNameDescriptionField: React.FC<K8sNameDescriptionFieldProps> = ({
           />
         </FormGroup>
       ) : null}
-    </FormSection>
+    </>
   );
+
+  return sectionless ? contents : <FormSection style={{ margin: 0 }}>{contents}</FormSection>;
```

Comment:
I think it makes sense to drop the section entirely and let the parent manage section considering we previously already removed the margin from the section.